src/openrct2/world/footpath.c

```diff
@@ -493,7 +506,8 @@ void game_command_place_footpath(sint32 *eax, sint32 *ebx, sint32 *ecx, sint32 *
         *edx & 0xFF,
         (*ebx >> 8) & 0xFF,
         *ebx & 0xFF,
-        *edi & 0xFF
+        *edi & 0xFF,
+        *ebp
```

Comment:
Follow the convention of the other fields. The game command function should split up the register into the two `uint8` parameters.